### DISCUSSION TITLE
[Auth0 client] Authorization code grant type

### DISCUSSION THREAD

**Discussion by georgev-97** [mia-platform/community#371] [Q&A] [answered]
### Which section?

Console

### Which is the version?

12

### Description

Hello guys,
I am trying to reach the `GET /authorize` of the Auth0-client from postman but I get this error:
`NOT FOUND GET /?error=access_denied&error_description=Service%20not%20found%3A%20https%3A%2F%2F_[MY-ENDPOINT]_%2F&state=80cdb9c3-f4a9-4fe1-adb3-536dfc4338b3`.

I've set the endpoint correctly and also configured the grant type in my auth0 tenant and the other endpoints like `/oauth/token` work just fine.

I've tried reaching the `GET /authorize` directly on Auth0 and it works

**Comment by malta895** [comment, marked as answer]
Hi @georgev-97,
That is the error returned by Auth0 when you are trying to use an audience which is not configured correctly on your Auth0 tenant.
Are you sure that the `audience` in your auth0-client configuration matches exactly the unique identifier of the API configured on your Auth0 tenant, as required by [Auth0 docs](https://auth0.com/docs/api/authentication#authorize-application)? 
What `audience` are you using in the direct `/authorize` call to Auth0?

**Reply by georgev-97** [reply]
Thank you, the documentation is a bit misleading since in the example the audience is set as follows:` "audience": "https://my-host.test.mia-platform.eu"`, while the correct one to set is a `*.auth0.com` URL.

I fixed by setting the audience as the identifier of my Auth0 API, in my direct call to Auth0 I didn't specify any audience, it is not a required field.

**Reply by malta895** [reply]
Thanks for your feedback! We'll check our documentation and improve it!

Anyways the `audience` field is not a required field in the `auth0-client` configuration either; if you don't configure it or leave it blank it won't be passed to the underlying Auth0 `/authorize` endpoint.

**Reply by georgev-97** [reply]
Thanks, that is what I need!
